test/integration/notifications/notification.spec.ts

```diff
@@ -49,24 +49,30 @@ describe("notification", () => {
       expect(sendNotification.mock.calls.length).toBeGreaterThan(0)
 
       let usersWithDeviceTokens = 0
-      for (const { ownerId } of activeAccounts) {
-        const user = await UsersRepository().findById(ownerId)
-        if (user instanceof Error) throw user
+      for (const { kratosUserId } of activeAccounts) {
+        const user = await UsersRepository().findById(kratosUserId)
+        if (user instanceof Error) return user
```

Comment:
Why return here? We usually throw from tests to stop execution when an unexpected error happens